sale_order_carrier_auto_assign/models/sale_order.py

```diff
@@ -1,17 +1,14 @@
 # Copyright 2020 Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
 from odoo import models
-from odoo.tools import config
 
 
 class SaleOrder(models.Model):
 
     _inherit = "sale.order"
 
     def action_confirm(self):
-        if config["test_enable"] and not self.env.context.get(
```

Comment:
this is changing the default behavior where ppl expects this to be working OOTB.
It requires an upgrade step to enable it by default on existing installations.